### ISSUE TITLE
Cannot read property 'wysiwyg' of undefined

### ISSUE THREAD

**Issue by Jenny888** [pencilblue/pencilblue#926]
Hi, I have just installed pencilblue locally. I tried to create new page and I got the message "The form is incomplete". When I opened the console I discovered I get this error on page load:

TypeError: Cannot read property 'wysiwyg' of undefined
    at link (http://localhost:8080/js/angular/directives/wysiwyg.js:24:20)
    at http://ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular.min.js:62:364
    at J (http://ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular.min.js:52:492)
    at http://ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular.min.js:59:302
    at http://ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular.min.js:70:373
    at H (http://ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular.min.js:97:5)
    at H (http://ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular.min.js:97:5)
    at http://ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular.min.js:98:173
    at h.$eval (http://ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular.min.js:108:40)
    at h.$digest (http://ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular.min.js:105:331) <div layout="layout" media="page.page_media" class="ng-isolate-scope">

P.S. Tried to switch libraries between CDN and Bower as was sujested in one similar issue here but it didnt help.

**Comment by Jenny888** [comment]
I searched a little more into this issue. Here is the code giving error because the variable <b>loc</b> is not defined.

/public/js/angular/directives/wysiwyg.js:

```javascript
   23          scope.availableElements = [{
   24:           name: loc.wysiwyg.NORMAL_TEXT,
   25            type: 'p'
   26          }, {
   27:           name: loc.wysiwyg.QUOTE,
   28            type: 'blockquote'
   29          }, {
   30:           name: loc.wysiwyg.PRE,
   31            type: 'pre'
   32          }, {
   33:           name: loc.wysiwyg.HEADING_1,
   34            type: 'h1'
   35          }, {
   36:           name: loc.wysiwyg.HEADING_2,
   37            type: 'h2'
   38          }, {
   39:           name: loc.wysiwyg.HEADING_3,
   40            type: 'h3'
   41          }, {
   42:           name: loc.wysiwyg.HEADING_4,
   43            type: 'h4'
   44          }, {
   45:           name: loc.wysiwyg.HEADING_5,
   46            type: 'h5'
   47          }, {
   48:           name: loc.wysiwyg.HEADING_6,
   49            type: 'h6'
   50          }];
   ..
  363          editableDiv.on('mouseup', function(event) {
  364            if(!scope.wysiwyg.layout.length) {
  365:             scope.setElement(loc.wysiwyg.NORMAL_TEXT, 'p');
  366            }
```

**Comment by brianhyder** [comment]
@Jenny888 Thank you for the report.  Can you provide the following information:

* version of PB that you are using?
* The active theme

@blakecallens The referenced code block appears in 0.5.0 too.  Need to verify that the localization script is getting brought in.

**Comment by blakecallens** [comment]
I'm assuming there's some sort of extraneous circumstance driving this. If pages were unable to be saved, across the board in 0.4.x and 0.5.0, we would have heard about it by now.

**Comment by blakecallens** [comment]
@brianhyder could it have to do with #921?

**Comment by blakecallens** [comment]
@Jenny888 are there any console errors before you try to save?

**Comment by brianhyder** [comment]
@blakecallens No, the referenced bug, #921 would have only prevented the ZN locale from loading.  No other side-effects.

**Comment by Jenny888** [comment]
Sorry for the delayed response.

My PB version is 0.4.1.
The active theme is PencilBlue.

Here is the console:

```
pbctrl install new
PencilBlue: Site Name:  (My PencilBlue Site) My Site
PencilBlue: Site Root:  (http://localhost:8080) 
PencilBlue: Address to bind to:  (0.0.0.0) 
PencilBlue: Site Port:  (8080) 
PencilBlue: MongoDB URL:  (mongodb://127.0.0.1:27017/) 
PencilBlue: Database Name:  (pencilblue) newpencil
PencilBlue: Do you want to install Bower components?:  (y/N) y
Installing PencilBlue to /home/evgenia/Projects/TestCMS/NewPencilBlue/new/...
Cloning into 'new'...
Installing Node modules...
npm WARN engine locale@0.0.20: wanted: {"node":">0.8.x <=1.0.0"} (current: {"node":"5.3.0","npm":"3.3.12"})

> kerberos@0.0.17 install /home/evgenia/Projects/TestCMS/NewPencilBlue/new/node_modules/kerberos
> (node-gyp rebuild) || (exit 0)

make: Entering directory '/home/evgenia/Projects/TestCMS/NewPencilBlue/new/node_modules/kerberos/build'
  CXX(target) Release/obj.target/kerberos/lib/kerberos.o
  CXX(target) Release/obj.target/kerberos/lib/worker.o
  CC(target) Release/obj.target/kerberos/lib/kerberosgss.o
../lib/kerberosgss.c:36:0: warning: ignoring #pragma clang diagnostic [-Wunknown-pragmas]
 #pragma clang diagnostic push
 ^
../lib/kerberosgss.c:37:0: warning: ignoring #pragma clang diagnostic [-Wunknown-pragmas]
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
 ^
../lib/kerberosgss.c: In function ‘authenticate_gss_client_wrap’:
../lib/kerberosgss.c:362:19: warning: variable ‘server_conf_flags’ set but not used [-Wunused-but-set-variable]
   char buf[4096], server_conf_flags;
                   ^
../lib/kerberosgss.c: At top level:
../lib/kerberosgss.c:930:0: warning: ignoring #pragma clang diagnostic [-Wunknown-pragmas]
 #pragma clang diagnostic pop
 ^
  CC(target) Release/obj.target/kerberos/lib/base64.o
  CXX(target) Release/obj.target/kerberos/lib/kerberos_context.o
  SOLINK_MODULE(target) Release/obj.target/kerberos.node
  COPY Release/kerberos.node
make: Leaving directory '/home/evgenia/Projects/TestCMS/NewPencilBlue/new/node_modules/kerberos/build'

> bson-ext@0.1.13 install /home/evgenia/Projects/TestCMS/NewPencilBlue/new/node_modules/bson-ext
> (node-gyp rebuild 2> builderror.log) || (exit 0)

make: Entering directory '/home/evgenia/Projects/TestCMS/NewPencilBlue/new/node_modules/bson-ext/build'
  CXX(target) Release/obj.target/bson/ext/bson.o
  SOLINK_MODULE(target) Release/obj.target/bson.node
  COPY Release/bson.node
make: Leaving directory '/home/evgenia/Projects/TestCMS/NewPencilBlue/new/node_modules/bson-ext/build'
pencilblue@0.4.1 /home/evgenia/Projects/TestCMS/NewPencilBlue/new
├── async@0.9.0 
├─┬ async-eventemitter@0.2.2 
│ └── async@0.2.10 
├─┬ cookies@0.5.0 
│ └── keygrip@1.0.1 
├─┬ coveralls@2.11.2 
│ ├─┬ js-yaml@3.0.1 
│ │ ├─┬ argparse@0.1.16 
│ │ │ ├── underscore@1.7.0 
│ │ │ └── underscore.string@2.4.0 
│ │ └── esprima@1.0.4 
│ ├── lcov-parse@0.0.6 
│ ├── log-driver@1.2.4 
│ └─┬ request@2.40.0 
│   ├── aws-sign2@0.5.0 
│   ├── forever-agent@0.5.2 
│   ├─┬ form-data@0.1.4 
│   │ └─┬ combined-stream@0.0.7 
│   │   └── delayed-stream@0.0.5 
│   ├─┬ hawk@1.1.1 
│   │ ├── boom@0.4.2 
│   │ ├── cryptiles@0.2.2 
│   │ ├── hoek@0.9.1 
│   │ └── sntp@0.2.4 
│   ├─┬ http-signature@0.10.1 
│   │ ├── asn1@0.1.11 
│   │ ├── assert-plus@0.1.5 
│   │ └── ctype@0.5.3 
│   ├── json-stringify-safe@5.0.1 
│   ├── mime-types@1.0.2 
│   ├── oauth-sign@0.3.0 
│   ├── qs@1.0.2 
│   ├── stringstream@0.0.5 
│   ├── tough-cookie@2.2.1 
│   └── tunnel-agent@0.4.2 
├── fakeredis@0.3.0 
├── formidable@1.0.17 
├── htmlencode@0.0.4 
├─┬ istanbul@0.3.6 
│ ├── abbrev@1.0.7 
│ ├─┬ escodegen@1.3.3 
│ │ ├── esprima@1.1.1 
│ │ ├── estraverse@1.5.1 
│ │ ├── esutils@1.0.0 
│ │ └─┬ source-map@0.1.43 
│ │   └── amdefine@1.0.0 
│ ├── esprima@2.0.0 
│ ├─┬ fileset@0.1.8 
│ │ ├─┬ glob@3.2.11 
│ │ │ └── minimatch@0.3.0 
│ │ └─┬ minimatch@0.4.0 
│ │   ├── lru-cache@2.7.3 
│ │   └── sigmund@1.0.1 
│ ├─┬ handlebars@1.3.0 
│ │ ├── optimist@0.3.7 
│ │ └─┬ uglify-js@2.3.6 
│ │   └── async@0.2.10 
│ ├─┬ mkdirp@0.5.1 
│ │ └── minimist@0.0.8 
│ ├── nopt@3.0.6 
│ ├─┬ once@1.3.3 
│ │ └── wrappy@1.0.1 
│ ├── resolve@0.7.4 
│ ├── supports-color@1.2.1 
│ ├── which@1.0.9 
│ └── wordwrap@0.0.3 
├── locale@0.0.20 
├─┬ mocha@2.1.0 
│ ├── commander@2.3.0 
│ ├─┬ debug@2.0.0 
│ │ └── ms@0.6.2 
│ ├── diff@1.0.8 
│ ├── escape-string-regexp@1.0.2 
│ ├─┬ glob@3.2.3 
│ │ ├── graceful-fs@2.0.3 
│ │ ├── inherits@2.0.1 
│ │ └── minimatch@0.2.14 
│ ├── growl@1.8.1 
│ ├─┬ jade@0.26.3 
│ │ ├── commander@0.6.1 
│ │ └── mkdirp@0.3.0 
│ └── mkdirp@0.5.0 
├── mocha-lcov-reporter@0.0.1 
├─┬ mongodb@2.0.33 
│ ├─┬ mongodb-core@1.1.32 
│ │ ├─┬ bson@0.3.2 
│ │ │ └─┬ bson-ext@0.1.13 
│ │ │   └── bindings@1.2.1 
│ │ └─┬ kerberos@0.0.17 
│ │   └── nan@2.0.9 
│ └─┬ readable-stream@1.0.31 
│   ├── core-util-is@1.0.2 
│   ├── isarray@0.0.1 
│   └── string_decoder@0.10.31 
├── node-uuid@1.4.3 
├─┬ node.extend@1.1.3 
│ └── is@2.1.0 
├─┬ nodemailer@0.5.3 
│ ├─┬ mailcomposer@0.2.12 
│ │ ├─┬ dkim-signer@0.1.2 
│ │ │ └── punycode@1.2.4 
│ │ ├── follow-redirects@0.0.3 
│ │ ├── he@0.3.6 
│ │ ├── mime@1.2.11 
│ │ └─┬ mimelib@0.2.19 
│ │   ├── addressparser@0.3.2 
│ │   └─┬ encoding@0.1.12 
│ │     └── iconv-lite@0.4.13 
│ └─┬ simplesmtp@0.3.35 
│   ├── rai@0.1.12 
│   └── xoauth2@0.1.8 
├─┬ npm@2.10.1 
│ ├── inherits@2.0.1 
│ ├─┬ mkdirp@0.5.1 
│ │ └── minimist@0.0.8 
│ ├─┬ npm-registry-client@6.3.3
│ │ └─┬ concat-stream@1.4.8
│ │   └─┬ readable-stream@1.1.13
│ │     ├── isarray@0.0.1 
│ │     └── string_decoder@0.10.31 
│ ├─┬ npmlog@1.2.0
│ │ └─┬ are-we-there-yet@1.0.4
│ │   └─┬ readable-stream@1.1.13
│ │     ├── isarray@0.0.1 
│ │     └── string_decoder@0.10.31 
│ ├─┬ readable-stream@1.0.33
│ │ ├── isarray@0.0.1 
│ │ └── string_decoder@0.10.31 
│ ├─┬ request@2.55.0
│ │ ├── aws-sign2@0.5.0 
│ │ ├─┬ combined-stream@0.0.7 
│ │ │ └── delayed-stream@0.0.5 
│ │ ├─┬ form-data@0.2.0
│ │ │ └── async@0.9.0 
│ │ ├─┬ http-signature@0.10.1 
│ │ │ ├── asn1@0.1.11 
│ │ │ ├── assert-plus@0.1.5 
│ │ │ └── ctype@0.5.3 
│ │ ├── isstream@0.1.2 
│ │ └── node-uuid@1.4.3 
│ ├─┬ sha@1.3.0
│ │ └─┬ readable-stream@1.1.13
│ │   ├── isarray@0.0.1 
│ │   └── string_decoder@0.10.31 
│ └── wrappy@1.0.1 
├── process@0.10.1 
├── redis@0.12.1 
├─┬ sanitize-html@1.1.7 
│ ├── he@0.4.1 
│ ├─┬ htmlparser2@3.3.0 
│ │ ├── domelementtype@1.3.0 
│ │ ├── domhandler@2.1.0 
│ │ └── domutils@1.1.6 
│ └── lodash@2.4.2 
├── semver@4.3.3 
├─┬ should@5.0.1 
│ ├── should-equal@0.3.1 
│ ├── should-format@0.0.7 
│ └── should-type@0.0.4 
├── underscore@1.8.3 
└─┬ winston@0.9.0 
  ├── colors@1.0.3 
  ├── cycle@1.0.3 
  ├── eyes@0.1.8 
  ├── isstream@0.1.2 
  ├── pkginfo@0.3.1 
  └── stack-trace@0.0.9 

npm WARN EPACKAGEJSON pencilblue@0.4.1 No license field.
Retrieving Bower components...
bower jqueryui-touch-punch#*       not-cached git://github.com/furf/jquery-ui-touch-punch.git#*
bower jqueryui-touch-punch#*          resolve git://github.com/furf/jquery-ui-touch-punch.git#*
bower jquery.spinjs#*              not-cached git://github.com/tdoherty/jQuery.SpinJS.git#*
bower jquery.spinjs#*                 resolve git://github.com/tdoherty/jQuery.SpinJS.git#*
bower angular-route#~1.3.2             cached git://github.com/angular/bower-angular-route.git#1.3.20
bower angular-route#~1.3.2           validate 1.3.20 against git://github.com/angular/bower-angular-route.git#~1.3.2
bower angular#~1.3.2                   cached git://github.com/angular/bower-angular.git#1.3.20
bower angular#~1.3.2                 validate 1.3.20 against git://github.com/angular/bower-angular.git#~1.3.2
bower jquery.iframe-transport#~1.0.0   cached git://github.com/cmlenz/jquery-iframe-transport.git#1.0.0
bower jquery.iframe-transport#~1.0.0 validate 1.0.0 against git://github.com/cmlenz/jquery-iframe-transport.git#~1.0.0
bower datetimepicker#~2.2.8            cached git://github.com/xdan/datetimepicker.git#2.2.9
bower datetimepicker#~2.2.8          validate 2.2.9 against git://github.com/xdan/datetimepicker.git#~2.2.8
bower respond#~1.4.2                   cached git://github.com/scottjehl/Respond.git#1.4.2
bower respond#~1.4.2                 validate 1.4.2 against git://github.com/scottjehl/Respond.git#~1.4.2
bower jqueryui#~1.11.0                 cached git://github.com/components/jqueryui.git#1.11.4
bower jqueryui#~1.11.0               validate 1.11.4 against git://github.com/components/jqueryui.git#~1.11.0
bower to-markdown#~0.0.2               cached git://github.com/domchristie/to-markdown.git#0.0.3
bower to-markdown#~0.0.2             validate 0.0.3 against git://github.com/domchristie/to-markdown.git#~0.0.2
bower markdown#~0.5.0                  cached git://github.com/evilstreak/markdown-js.git#0.5.0
bower markdown#~0.5.0                validate 0.5.0 against git://github.com/evilstreak/markdown-js.git#~0.5.0
bower ng-sortable#~1.1.6               cached git://github.com/a5hik/ng-sortable.git#1.1.9
bower ng-sortable#~1.1.6             validate 1.1.9 against git://github.com/a5hik/ng-sortable.git#~1.1.6
bower rangy-release#~1.3.0             cached git://github.com/timdown/rangy-release.git#1.3.0
bower rangy-release#~1.3.0           validate 1.3.0 against git://github.com/timdown/rangy-release.git#~1.3.0
bower fontawesome#~4.2.0               cached git://github.com/FortAwesome/Font-Awesome.git#4.2.0
bower fontawesome#~4.2.0             validate 4.2.0 against git://github.com/FortAwesome/Font-Awesome.git#~4.2.0
bower jquery.validation#~1.13.0        cached git://github.com/jzaefferer/jquery-validation.git#1.13.1
bower jquery.validation#~1.13.0      validate 1.13.1 against git://github.com/jzaefferer/jquery-validation.git#~1.13.0
bower jquery#~1.11.1                   cached git://github.com/jquery/jquery-dist.git#1.11.3
bower jquery#~1.11.1                 validate 1.11.3 against git://github.com/jquery/jquery-dist.git#~1.11.1
bower angular-sanitize#~1.3.2          cached git://github.com/angular/bower-angular-sanitize.git#1.3.20
bower angular-sanitize#~1.3.2        validate 1.3.20 against git://github.com/angular/bower-angular-sanitize.git#~1.3.2
bower jquery-file-upload#~9.7.0        cached git://github.com/blueimp/jQuery-File-Upload.git#9.7.2
bower jquery-file-upload#~9.7.0      validate 9.7.2 against git://github.com/blueimp/jQuery-File-Upload.git#~9.7.0
bower html5shiv#~3.7.2                 cached git://github.com/aFarkas/html5shiv.git#3.7.3
bower html5shiv#~3.7.2               validate 3.7.3 against git://github.com/aFarkas/html5shiv.git#~3.7.2
bower danialfarid-angular-file-upload#~1.6.12           cached git://github.com/danialfarid/angular-file-upload.git#1.6.12
bower danialfarid-angular-file-upload#~1.6.12         validate 1.6.12 against git://github.com/danialfarid/angular-file-upload.git#~1.6.12
bower bootstrap#~3.1.1                                  cached git://github.com/twbs/bootstrap.git#3.1.1
bower bootstrap#~3.1.1                                validate 3.1.1 against git://github.com/twbs/bootstrap.git#~3.1.1
bower jquery#>=1.6                                      cached git://github.com/jquery/jquery-dist.git#2.2.0
bower jquery#>=1.6                                    validate 2.2.0 against git://github.com/jquery/jquery-dist.git#>=1.6
bower jquery#>= 1.7.2                                   cached git://github.com/jquery/jquery-dist.git#2.2.0
bower jquery#>= 1.7.2                                 validate 2.2.0 against git://github.com/jquery/jquery-dist.git#>= 1.7.2
bower jqueryui-touch-punch#*                          checkout master
bower jquery.spinjs#*                                 checkout master
bower jquery#>= 1.6.4                                   cached git://github.com/jquery/jquery-dist.git#2.2.0
bower jquery#>= 1.6.4                                 validate 2.2.0 against git://github.com/jquery/jquery-dist.git#>= 1.6.4
bower he#~0.4.1                                         cached git://github.com/mathiasbynens/he.git#0.4.1
bower he#~0.4.1                                       validate 0.4.1 against git://github.com/mathiasbynens/he.git#~0.4.1
bower blueimp-tmpl#>=2.5.3                              cached git://github.com/blueimp/JavaScript-Templates.git#3.1.0
bower blueimp-tmpl#>=2.5.3                            validate 3.1.0 against git://github.com/blueimp/JavaScript-Templates.git#>=2.5.3
bower blueimp-canvas-to-blob#>=2.1.0                    cached git://github.com/blueimp/JavaScript-Canvas-to-Blob.git#3.1.0
bower blueimp-canvas-to-blob#>=2.1.0                  validate 3.1.0 against git://github.com/blueimp/JavaScript-Canvas-to-Blob.git#>=2.1.0
bower blueimp-load-image#>=1.11.0                       cached git://github.com/blueimp/JavaScript-Load-Image.git#2.1.0
bower blueimp-load-image#>=1.11.0                     validate 2.1.0 against git://github.com/blueimp/JavaScript-Load-Image.git#>=1.11.0
bower jqueryui-touch-punch#*                          resolved git://github.com/furf/jquery-ui-touch-punch.git#4bc0091452
bower jquery-ui#>=1.8                                   cached git://github.com/components/jqueryui.git#1.11.4
bower jquery-ui#>=1.8                                 validate 1.11.4 against git://github.com/components/jqueryui.git#>=1.8
bower jquery.spinjs#*                                 resolved git://github.com/tdoherty/jQuery.SpinJS.git#8c4d107ae9
bower spin.js#*                                         cached git://github.com/fgnass/spin.js.git#2.3.2
bower spin.js#*                                       validate 2.3.2 against git://github.com/fgnass/spin.js.git#*
bower jquery.iframe-transport#~1.0.0                   install jquery.iframe-transport#1.0.0
bower respond#~1.4.2                                   install respond#1.4.2
bower datetimepicker#~2.2.8                            install datetimepicker#2.2.9
bower jqueryui#~1.11.0                                 install jqueryui#1.11.4
bower to-markdown#~0.0.2                               install to-markdown#0.0.3
bower angular-route#~1.3.2                             install angular-route#1.3.20
bower fontawesome#~4.2.0                               install fontawesome#4.2.0
bower rangy-release#~1.3.0                             install rangy-release#1.3.0
bower markdown#~0.5.0                                  install markdown#0.5.0
bower ng-sortable#~1.1.6                               install ng-sortable#1.1.9
bower angular#~1.3.2                                   install angular#1.3.20
bower html5shiv#~3.7.2                                 install html5shiv#3.7.3
bower jquery.validation#~1.13.0                        install jquery.validation#1.13.1
bower jquery-file-upload#~9.7.0                        install jquery-file-upload#9.7.2
bower jquery#~1.11.1                                   install jquery#1.11.3
bower angular-sanitize#~1.3.2                          install angular-sanitize#1.3.20
bower bootstrap#~3.1.1                                 install bootstrap#3.1.1
bower danialfarid-angular-file-upload#~1.6.12          install danialfarid-angular-file-upload#1.6.12
bower he#~0.4.1                                        install he#0.4.1
bower blueimp-tmpl#>=2.5.3                             install blueimp-tmpl#3.1.0
bower blueimp-canvas-to-blob#>=2.1.0                   install blueimp-canvas-to-blob#3.1.0
bower blueimp-load-image#>=1.11.0                      install blueimp-load-image#2.1.0
bower jqueryui-touch-punch#*                           install jqueryui-touch-punch#4bc0091452
bower jquery-ui#>=1.8                                  install jquery-ui#1.11.4
bower jquery.spinjs#*                                  install jquery.spinjs#8c4d107ae9
bower spin.js#*                                        install spin.js#2.3.2

jquery.iframe-transport#1.0.0 public/bower_components/jquery.iframe-transport
└── jquery#1.11.3

respond#1.4.2 public/bower_components/respond

datetimepicker#2.2.9 public/bower_components/datetimepicker
└── jquery#1.11.3

jqueryui#1.11.4 public/bower_components/jqueryui
└── jquery#1.11.3

to-markdown#0.0.3 public/bower_components/to-markdown
└── he#0.4.1

angular-route#1.3.20 public/bower_components/angular-route
└── angular#1.3.20

fontawesome#4.2.0 public/bower_components/fontawesome

rangy-release#1.3.0 public/bower_components/rangy-release

markdown#0.5.0 public/bower_components/markdown

ng-sortable#1.1.9 public/bower_components/ng-sortable
└── angular#1.3.20

angular#1.3.20 public/bower_components/angular

html5shiv#3.7.3 public/bower_components/html5shiv

jquery.validation#1.13.1 public/bower_components/jquery.validation
└── jquery#1.11.3

jquery-file-upload#9.7.2 public/bower_components/jquery-file-upload
├── blueimp-canvas-to-blob#3.1.0
├── blueimp-load-image#2.1.0
├── blueimp-tmpl#3.1.0
└── jquery#1.11.3

jquery#1.11.3 public/bower_components/jquery

angular-sanitize#1.3.20 public/bower_components/angular-sanitize
└── angular#1.3.20

bootstrap#3.1.1 public/bower_components/bootstrap
└── jquery#1.11.3

danialfarid-angular-file-upload#1.6.12 public/bower_components/danialfarid-angular-file-upload

he#0.4.1 public/bower_components/he

blueimp-tmpl#3.1.0 public/bower_components/blueimp-tmpl

blueimp-canvas-to-blob#3.1.0 public/bower_components/blueimp-canvas-to-blob

blueimp-load-image#2.1.0 public/bower_components/blueimp-load-image

jqueryui-touch-punch#4bc0091452 public/bower_components/jqueryui-touch-punch
├── jquery#1.11.3
└── jquery-ui#1.11.4

jquery-ui#1.11.4 public/bower_components/jquery-ui
└── jquery#1.11.3

jquery.spinjs#8c4d107ae9 public/bower_components/jquery.spinjs
├── jquery#1.11.3
└── spin.js#2.3.2

spin.js#2.3.2 public/bower_components/spin.js
Creating config.js...
```

**Comment by Jenny888** [comment]
Here is the console when I start

```
nodemon pencilblue.js
[nodemon] 1.8.1
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `node pencilblue.js`
No overrides are available, skipping to defaults after searching the following files:
* /home/evgenia/Projects/TestCMS/pencilblue/pencilblue-master/config.js
* /etc/pencilblue/config.js
2016-01-18T08:31:07.107Z - info: [M] SystemStartup: Log Level is: info
2016-01-18T08:31:11.289Z - info: [M] System[M]: 1 workers spawned. Listening for disconnects.
No overrides are available, skipping to defaults after searching the following files:
* /home/evgenia/Projects/TestCMS/pencilblue/pencilblue-master/config.js
* /etc/pencilblue/config.js
2016-01-18T08:31:12.457Z - info: [1] SystemStartup: Log Level is: info
2016-01-18T08:31:15.246Z - info: [1] PencilBlue: Ensuring indices...
2016-01-18T08:31:15.505Z - info: [1] PencilBlue: pencilblue running at site root [http://localhost:8080] on port [8080]
2016-01-18T08:31:15.711Z - info: [1] PencilBlue: Ready to run!
2016-01-18T08:31:38.318Z - warn: [1] PencilBlue: pb.users is deprecated.  Use new pb.UserService(context) instead
```

And here is the error from debugger when I load New Page and new Article pages

```javascript
TypeError: Cannot read property 'wysiwyg' of undefined
    at link (http://localhost:8080/js/angular/directives/wysiwyg.js:24:20)
    at http://ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular.min.js:62:364
    at J (http://ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular.min.js:52:492)
    at http://ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular.min.js:59:302
    at http://ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular.min.js:70:373
    at H (http://ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular.min.js:97:5)
    at H (http://ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular.min.js:97:5)
    at http://ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular.min.js:98:173
    at h.$eval (http://ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular.min.js:108:40)
    at h.$digest (http://ajax.googleapis.com/ajax/libs/angularjs/1.2.15/angular.min.js:105:331) <div layout="layout" media="page.page_media" class="ng-isolate-scope">
```

The others pages work properly as I have seen.

**Comment by Jenny888** [comment]
Same error on Manage Users page

```
TypeError: Cannot read property 'generic' of undefined
    at new <anonymous> (users:217)
    at d (angular.js:3805)
    at Object.instantiate (angular.js:3816)
    at angular.js:6922
    at angular.js:6309
    at r (angular.js:330)
    at J (angular.js:6296)
    at h (angular.js:5761)
    at h (angular.js:5764)
    at angular.js:5666
```

```javascript
$scope.tableHeaders = [{
			name: loc.generic.USERNAME,
			field: 'username',
			sortAsc: false,
			sortDesc: false
		} ...
```
P.S. I am new to this so I dont know if I am doing something wrong but it seems the <strong>loc</strong> variable is not working for me.

**Comment by Jenny888** [comment]
I finally found the solution. The problem was that my local system lang is el_GR.UTF-8 and there is no file for GR to initialize the <b>loc</b> var. 

I solved the problem by adding a "el-GR.js" file in "public/localization" folder.

Everything is working properly now. 

I didnt check all the code but if you haven't done that, I would suggest to add check for system lang and if  there isnt a .js file to initialize loc, to assign default values.

Thank you for your time and your help :thumbsup:

**Comment by brianhyder** [comment]
@Jenny888 Thank you for sending over the info.  We'll get that fixed.

**Comment by brianhyder** [comment]
Leaving open until we get a fix in.

**Comment by brianhyder** [comment]
@Jenny888 Just wanted to followup with you.  The issue has been fixed in the 0.5.0 branch.  Thank you again for reporting the issue.
